### DISCUSSION TITLE
Request to Launch Visual Studio Code for Android Devices

### DISCUSSION THREAD

**Discussion by Shriradhakrishnasharnam** [microsoft/vscode-discussions#1661] [Extension Development QnA] [answered]
Respected Visual Studio Code Team,

I trust this message finds you in good spirits and well.

As an avid developer and a long-time user of Visual Studio Code, I am reaching out to request the release of a dedicated VS Code application for Android devices. With the rapid advancements in mobile technology, Android smartphones now come equipped with powerful RAM, high-performance processors, and even dual-screen or foldable designs. This has effectively minimized the gap between laptops and mobile devices, making it more feasible to carry out professional development tasks on the go.

Currently, there are mobile code editors like Spck Code Editor on the Play Store, which provide a commendable coding experience. However, as a developer who is deeply familiar with and reliant on Visual Studio Code, having a native Android version would enhance productivity significantly, especially when developing apps, extensions, and web applications while mobile.

I kindly request you to consider releasing a VS Code app for Android. This would benefit many developers who, like me, are increasingly leveraging mobile devices for development work due to their enhanced capabilities.

Thank you for your time and consideration. I look forward to your positive response.

Warm regards,
B R Shriradhakrishnasharnam

**Comment by gjsjohnmurray** [comment, marked as answer]
Please see [this pinned READ FIRST thread](https://github.com/microsoft/vscode-discussions/discussions/5) to learn about the focus of this discussion forum. It also points to other locations where your request could be made (after first checking for duplicates).

Thanks for your cooperation.

**Comment by olguzzar** [comment]
As answered above, this forum is specifically for extension authors. There is currently an [open issue](https://github.com/microsoft/vscode/issues/206267) on the main VS Code repo tracking a request for an Android app that you can follow. Thanks!
